libraries/redcore/layouts/webservice/documentation.php

```diff
@@ -67,6 +67,8 @@ function printWindow() {
 				<strong><?php echo JText::_('LIB_REDCORE_API_HAL_WEBSERVICE_DOCUMENTATION_WSDL_ACCESS_URL'); ?></strong>:
 				<?php echo RApiHalHelper::buildWebserviceFullUrl($view->client, $view->webserviceName, $view->webserviceVersion, 'soap') . '&wsdl'; ?><br />
 			<?php endif; ?>
+			<strong><?php echo JText::_('LIB_REDCORE_API_HAL_WEBSERVICE_DOCUMENTATION_DOCUMENTATION_URL'); ?></strong>:
+			<?php echo RApiHalHelper::buildWebserviceFullUrl($view->client, $view->webserviceName, $view->webserviceVersion, 'hal', 'doc'); ?><br />
```

Comment:
I have a note in my notebook about setting those links smaller + making them links not just text that opens in new window, maybe you could put it in this PR if you are willing :)